quinn-proto/src/tests.rs

```diff
@@ -192,9 +190,11 @@ impl Pair {
                 socket.send_to(&x.packet, x.destination).unwrap();
             }
             if self.server.addr == x.destination {
-                self.server
-                    .inbound
-                    .push_back((self.time + self.latency, x.ecn, x.packet));
+                self.server.inbound.push_back((
+                    self.time + Duration::from_micros(self.latency),
```

Comment:
Perhaps `self.latency` should be a `Duration`?

Reply:
Yup, good call.